source/docs/software/vscode-overview/importing-gradle-project.rst

```diff
@@ -1,12 +1,17 @@
 Importing a Gradle Project
 ==========================
 
-Due to changes in the project, it is necessary to update the build files for a previous years Gradle project. To make it easy for teams to import previous years gradle projects into the current year's framework, WPILib includes a wizard for importing previous years projects into VS Code. This will generate the necessary gradle components and load the project into VS Code. In place upgrades are not supported. The importer automatically imports the old command framework 3rd party library.
+Due to changes in the project, it is necessary to update the build files for a previous years Gradle project.
+
+Automatic import
+----------------
+
+To make it easy for teams to import previous years gradle projects into the current year's framework, WPILib includes a wizard for importing previous years projects into VS Code. This will generate the necessary gradle components and load the project into VS Code. In place upgrades are not supported. The importer automatically imports the old command framework 3rd party library.
```

Comment:
Unrelated to this PR, but do we still want this to be the intended behavior?

Reply:
We actually changed just changed this, will update the PR